src/types/keywords.ts

```diff
@@ -0,0 +1,24 @@
+import { ParsedUrlQueryInput } from 'querystring';
+import { Movie } from '.';
+
+export interface KeywordsOptions extends ParsedUrlQueryInput {
+	include_adult?: boolean;
+	language?: string;
+}
+
+export interface BelongingMovies{
+  page: number;
+	results: Movie[];
```

Comment:
Some extra indenting in `BelongingMovies`, `KeywordsOptions`, `LanguageOption` and `PageOption`